package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azuredevops-test-reporter-ts",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
@sundarlogan we should start thinking of having semantic version

Reply:
yes, added it.